<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="16699" started-at="2018-08-12T17:31:55Z" finished-at="2018-08-12T17:32:12Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="16699" started-at="2018-08-12T17:31:55Z" finished-at="2018-08-12T17:32:12Z">
      <class name="com.testCases.TestLogin">
        <test-method status="PASS" signature="actiTimeLoginTest(com.pojo.UserCredentials)[pri:0, instance:com.testCases.TestLogin@29ee9faa]" name="actiTimeLoginTest" duration-ms="8063" started-at="2018-08-12T17:31:55Z" data-provider="loginCredentialsDP" finished-at="2018-08-12T17:32:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[com.pojo.UserCredentials@5a8e6209]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- actiTimeLoginTest -->
        <test-method status="FAIL" signature="actiTimeLoginTest(com.pojo.UserCredentials)[pri:0, instance:com.testCases.TestLogin@29ee9faa]" name="actiTimeLoginTest" duration-ms="8578" started-at="2018-08-12T17:32:04Z" data-provider="loginCredentialsDP" finished-at="2018-08-12T17:32:12Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[com.pojo.UserCredentials@1caa0244]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The user credentials are not valid expected [0] but found [1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The user credentials are not valid expected [0] but found [1]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.testCases.TestLogin.actiTimeLoginTest(TestLogin.java:39)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- actiTimeLoginTest -->
      </class> <!-- com.testCases.TestLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
